maintainers/maintainer-list.nix

```
@@ -761,6 +761,11 @@
     githubId = 786394;
     name = "Alexander Krupenkin ";
   };
+  akshayk = {
+    github = "akshayk ";
```

```suggestion
  akshayka = {
    github = "akshayka";
```

I think that's the correct name